plugins/sel_length.lua

```diff
@@ -0,0 +1,40 @@
+-- mod-version:3
+-- Selection length plugin
+-- Shows the selection length on statusbar
+local core = require "core"
+local StatusView = require "core.statusview"
+
+core.status_view:add_item({
+	predicate = StatusView.predicate_docview,
+	name = "doc:sel_length",
+	alignment = StatusView.Item.LEFT,
+	get_item = function()
+		local dv = core.active_view
+		-- To fix crash
+		-- e.g. search_ui dir selection causes crash otherwise
+		if dv.doc == nil then return {} end
```

Comment:
This was happening because the predicate was `nil`. With a working predicate this won't be needed anymore.